iocage/lib/Jail.py

```diff
@@ -1420,10 +1430,7 @@ def _launch_persistent_jail(
             passthru=passthru
         )
         if returncode > 0:
```

Comment:
please also change it here.

Reply:
we have more of that here:

```
iocage/lib/JailState.py:112:        if returncode > 0:
iocage/lib/JailState.py:136:        if returncode > 0:
iocage/lib/JailState.py:214:        if returncode > 0:
iocage/lib/JailState.py:235:        if returncode > 0:
iocage/lib/Jail.py:1422:        if returncode > 0:
iocage/lib/Jail.py:1515:        if returncode > 0:
iocage/lib/helpers.py:175:    if returncode > 0:
```

i'm not being paranoid here, or?

Reply:
Good catch. We evaluate the error codes at the calling functions. The premature exist (still returning the `stdout`, `stderr`, `returncode` tuple) is just to leave the function early.